File: AppGenerateFiles/dbo/Model/Catalogo_Clientes.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Catalogo_Clientes : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? codigo_cliente { get; set; }
       public string? primer_nombre { get; set; }
       public string? segundo_nombre { get; set; }
       public string? primer_apellido { get; set; }
       public string? segundo_apellidio { get; set; }
       public int? id_tipo_identificacion { get; set; }
       public string? identificacion { get; set; }
       public string? sexo { get; set; }
       public DateTime? fecha_nacimiento { get; set; }
       public int? id_profesion { get; set; }
       public int? id_departamento { get; set; }
       public int? id_municipio { get; set; }
       public string? correo { get; set; }
       public string? telefono { get; set; }
       public string? direccion { get; set; }
       public string? hora { get; set; }
       public DateTime? fecha { get; set; }
       public string? observaciones { get; set; }
       public string? estado_civil { get; set; }
       public int? tipoc { get; set; }
       public string? tipo_firma { get; set; }
       public string? valor_cliente { get; set; }
       public string? operadora_celular { get; set; }
       public Double? valor_interes { get; set; }
       public string? solo_acreedor { get; set; }
       public Double? promedio { get; set; }
       public int? id_clasificacion { get; set; }
       public int? id_clasificacion_interes { get; set; }
       [ManyToOne(TableName = "Catalogo_Clasificacion_Cliente", KeyColumn = "id_clasificacion", ForeignKeyColumn = "id_clasificacion")]
       public Catalogo_Clasificacion_Cliente? Catalogo_Clasificacion_Cliente { get; set; }
       [ManyToOne(TableName = "Catalogo_Clasificacion_Interes", KeyColumn = "id_clasificacion_interes", ForeignKeyColumn = "id_clasificacion_interes")]
       public Catalogo_Clasificacion_Interes? Catalogo_Clasificacion_Interes { get; set; }
       [ManyToOne(TableName = "Catalogo_Departamento", KeyColumn = "id_departamento", ForeignKeyColumn = "id_departamento")]
       public Catalogo_Departamento? Catalogo_Departamento { get; set; }
       [ManyToOne(TableName = "Catalogo_Municipio", KeyColumn = "id_municipio", ForeignKeyColumn = "id_municipio")]
       public Catalogo_Municipio? Catalogo_Municipio { get; set; }
       [ManyToOne(TableName = "Catalogo_Profesiones", KeyColumn = "id_profesion", ForeignKeyColumn = "id_profesion")]
       public Catalogo_Profesiones? Catalogo_Profesiones { get; set; }
       [ManyToOne(TableName = "Catalogo_Tipo_Identificacion", KeyColumn = "id_tipo_identificacion", ForeignKeyColumn = "id_tipo_identificacion")]
       public Catalogo_Tipo_Identificacion? Catalogo_Tipo_Identificacion { get; set; }
       [OneToMany(TableName = "Condicion_Laboral_Cliente", KeyColumn = "codigo_cliente", ForeignKeyColumn = "id_cliente")]
       public List<Condicion_Laboral_Cliente>? Condicion_Laboral_Cliente { get; set; }
       [OneToMany(TableName = "Transaction_Contratos", KeyColumn = "codigo_cliente", ForeignKeyColumn = "codigo_cliente")]
       public List<Transaction_Contratos>? Transaction_Contratos { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Catalogo_Clientes.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Clasificacion_Cliente }  from './Catalogo_Clasificacion_Cliente.js'
import { Catalogo_Clasificacion_Interes }  from './Catalogo_Clasificacion_Interes.js'
import { Catalogo_Departamento }  from './Catalogo_Departamento.js'
import { Catalogo_Municipio }  from './Catalogo_Municipio.js'
import { Catalogo_Profesiones }  from './Catalogo_Profesiones.js'
import { Catalogo_Tipo_Identificacion }  from './Catalogo_Tipo_Identificacion.js'
import { Condicion_Laboral_Cliente }  from './Condicion_Laboral_Cliente.js'
import { Transaction_Contratos }  from './Transaction_Contratos.js'
class Catalogo_Clientes extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ codigo_cliente;
   /**@type {String}*/ primer_nombre;
   /**@type {String}*/ segundo_nombre;
   /**@type {String}*/ primer_apellido;
   /**@type {String}*/ segundo_apellidio;
   /**@type {String}*/ identificacion;
   /**@type {String}*/ sexo;
   /**@type {Date}*/ fecha_nacimiento;
   /**@type {String}*/ correo;
   /**@type {String}*/ telefono;
   /**@type {String}*/ direccion;
   /**@type {String}*/ hora;
   /**@type {Date}*/ fecha;
   /**@type {String}*/ observaciones;
   /**@type {String}*/ estado_civil;
   /**@type {Number}*/ tipoc;
   /**@type {String}*/ tipo_firma;
   /**@type {String}*/ valor_cliente;
   /**@type {String}*/ operadora_celular;
   /**@type {Number}*/ valor_interes;
   /**@type {String}*/ solo_acreedor;
   /**@type {Number}*/ promedio;
   /**@type {Catalogo_Clasificacion_Cliente} ManyToOne*/ Catalogo_Clasificacion_Cliente;
   /**@type {Catalogo_Clasificacion_Interes} ManyToOne*/ Catalogo_Clasificacion_Interes;
   /**@type {Catalogo_Departamento} ManyToOne*/ Catalogo_Departamento;
   /**@type {Catalogo_Municipio} ManyToOne*/ Catalogo_Municipio;
   /**@type {Catalogo_Profesiones} ManyToOne*/ Catalogo_Profesiones;
   /**@type {Catalogo_Tipo_Identificacion} ManyToOne*/ Catalogo_Tipo_Identificacion;
   /**@type {Array<Condicion_Laboral_Cliente>} OneToMany*/ Condicion_Laboral_Cliente;
   /**@type {Array<Transaction_Contratos>} OneToMany*/ Transaction_Contratos;
}
export { Catalogo_Clientes }
